modules/home-environment.nix

```diff
@@ -40,11 +72,44 @@ let
         '';
       };
 
-      time = mkOption {
+      name = mkOption {
         default = null;
         type = types.nullOr types.str;
         description = ''
-          The language to use for formatting times.
+          The language to use for personal names.
+        '';
+      };
+
+
+      address = mkOption {
+        default = null;
+        type = types.nullOr types.str;
+        description = ''
+          The language to use for addresses.
+        '';
+      };
+
+      telephone = mkOption {
+        default = null;
+        type = types.nullOr types.str;
+        description = ''
+          The language to use for telephone numbers.
+        '';
+      };
+
+      measurement = mkOption {
+        default = null;
+        type = types.nullOr types.str;
+        description = ''
+          The language to use for measurement values.
+        '';
+      };
+
+      identification = mkOption {
```

Comment:
I think you can remove this one. As far as I can tell it's not something users are expected to set:

> LC_IDENTIFICATION - this is not a user-visible category, it contains information about the locale itself and is rarely useful for users or developers (but is listed here for completeness sake).

From https://sourceware.org/glibc/wiki/Locales#Overview

Reply:
you're right, I just took all the variables reported by `locale`